h/migrations/env.py

```diff
@@ -28,14 +27,13 @@
 
 
 def get_database_url(hint):
-    if 'DATABASE_URL' in os.environ:
-        urlparse.uses_netloc.append("postgres")
-        urlparse.uses_netloc.append("sqlite")
-        url = urlparse.urlparse(os.environ["DATABASE_URL"])
-        if url.scheme == 'postgres':
-            url.scheme = url.scheme + '+psycopg2'
-        return urlparse.urlunparse(url)
-    return hint
+    if 'DATABASE_URL' not in os.environ:
+        return hint
+
+    url = os.environ['DATABASE_URL']
+    if url.startswith('postgres://'):
```

Comment:
We're only really trying to deal with the fact that the Heroku-style URLs (`postgres://`) don't specify a connection adapter. We don't want to force correct sqlalchemy URLs to do something other than what they specify.